api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -1027,11 +1029,87 @@ def get_managed_venues(offerer_id: int) -> utils.BackofficeResponse:
+    form = pro_forms.CreateVenueWithoutSIRETForm(offerer)
+    return _render_get_create_venue_without_siret_form(form, offerer_id)
+
+
+@offerer_blueprint.route("/create-without-siret", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.CREATE_PRO_ENTITY)
+def create_venue(offerer_id: int) -> utils.BackofficeResponse:
+    offerer = db.session.query(offerers_models.Offerer).get_or_404(offerer_id)
+    form = pro_forms.CreateVenueWithoutSIRETForm(offerer)
+    if not form.validate():
+        mark_transaction_as_invalid()
+        return _render_get_create_venue_without_siret_form(form, offerer_id), 400
+
+    attachment_venue = offerers_api.get_venue_by_id(form.attachement_venue.data)
+    assert attachment_venue.offererAddress
+
+    address_body_model = offerers_schemas.AddressBodyModel(
+        street=offerers_schemas.VenueAddress(attachment_venue.offererAddress.address.street),
+        city=offerers_schemas.VenueCity(attachment_venue.offererAddress.address.city),
+        postalCode=offerers_schemas.VenuePostalCode(attachment_venue.offererAddress.address.postalCode),
+        inseeCode=offerers_schemas.VenueInseeCode(attachment_venue.offererAddress.address.inseeCode),
+        latitude=float(attachment_venue.offererAddress.address.latitude),
+        longitude=float(attachment_venue.offererAddress.address.longitude),
+        banId=attachment_venue.offererAddress.address.banId,
+        label=None,
+    )
+
+    venue_creation_info = venues_serialize.PostVenueBodyModel(
+        address=address_body_model,
+        comment=offerers_schemas.VenueComment("Lieu sans SIRET car dépend du SIRET d'un autre lieu"),
+        siret=None,
+        bookingEmail=offerers_schemas.VenueBookingEmail(attachment_venue.bookingEmail),
+        managingOffererId=offerer_id,
+        name=offerers_schemas.VenueName(form.public_name.data),
+        publicName=offerers_schemas.VenuePublicName(form.public_name.data),
+        venueLabelId=None,
+        venueTypeCode=attachment_venue.venueTypeCode.name,
+        withdrawalDetails=None,
+        description=None,
+        contact=None,
+        audioDisabilityCompliant=None,
+        mentalDisabilityCompliant=None,
+        motorDisabilityCompliant=None,
+        visualDisabilityCompliant=None,
+        isOpenToPublic=False,
+    )
+    offerer_address = offerers_api.create_offerer_address(
+        attachment_venue.managingOffererId, attachment_venue.offererAddress.address.id
+    )
+    venue = offerers_api.create_venue(venue_creation_info, current_user, offerer_address=offerer_address)
+    venue.isPermanent = True
+    db.session.add(venue)
+    offerers_api.link_venue_to_pricing_point(venue, attachment_venue.id)
+
+    flash(Markup("Le partenaire culturel <b>{name}</b> a été créé").format(name=venue.common_name), "success")
+    return redirect(url_for("backoffice_web.offerer.get", offerer_id=offerer_id), code=303)
```

Comment:
Au minimum, il faut revenir vers le même onglet : 
```suggestion
    return redirect(url_for("backoffice_web.offerer.get", offerer_id=offerer_id, active_tab="managed_venues"), code=303)
```
ou peut-être choisir d'afficher la page de la nouvelle Venue : 
```suggestion
    return redirect(url_for("backoffice_web.venue.get", venue_id=venue.id), code=303)
```